src/EditorFeatures/Core.Wpf/InlineHints/InlineParameterNameHintsFormatDefinition.cs

```diff
@@ -12,7 +12,7 @@
 using Microsoft.VisualStudio.Text.Classification;
 using Microsoft.VisualStudio.Utilities;
 
-namespace Microsoft.CodeAnalysis.Editor.InlineParameterNameHints
+namespace Microsoft.CodeAnalysis.Editor.InlineHints
```

Comment:
name is changed as this will be the general namespace for all hints in the future (i.e. if we show type-hints for 'var').